src/utils/navigation.ts

```diff
@@ -5,95 +5,109 @@ export const NAVIGATION: Navigation = {
   [config.SOLD_LAND_FILTER_URL]: {
     currentPage: config.SOLD_LAND_FILTER_PAGE,
     previousPage: config.LANDING_URL,
-    nextPage: config.SECURE_REGISTER_FILTER_URL
+    nextPage: [config.SECURE_REGISTER_FILTER_URL, config.CANNOT_USE_URL]
```

Comment:
Again, how does this change and the one below relate to the requirement of redirecting to the land sold page?

Reply:
Navigation will not be used for this story, it will be reverted